ta_services/keymaster/km.c

```diff
@@ -1,12 +1,45 @@
 /* SPDX-License-Identifier: BSD-2-Clause */
 /* Copyright (c) 2018, Linaro Limited */
 
+#include <pta_system.h>
 #include <tee_internal_api.h>
 
 #include "km.h"
 
-TEE_Result km_add_rng_entropy(const void __unused *buf, size_t __unused blen)
+TEE_Result km_add_rng_entropy(const void *buf, size_t blen)
 {
-	/* Stubbed until the system PTA is available */
-	return TEE_ERROR_NOT_IMPLEMENTED;
+	static const TEE_UUID system_uuid = PTA_SYSTEM_UUID;
+	TEE_TASessionHandle sess = TEE_HANDLE_NULL;
+	TEE_Result res;
+	uint32_t l_pts;
```

Comment:
Please drop the `l_` prefix here and the row below. Also I'd prefer `pt` over `pts`.

Reply:
fixed